packages/manager/src/components/SelectRegionPanel/RegionHelperText.tsx

```diff
@@ -14,8 +14,12 @@ export const RegionHelperText = (props: RegionHelperTextProps) => {
   const { onClick, showCoreHelperText, ...rest } = props;
 
   return (
-    <Box {...rest}>
-      <Typography data-testid="region-select-helper-test" variant="body1">
+    <Box component="span" sx={{ display: 'block' }} {...rest}>
```

Comment:
optional - not that I think it's likely to happen, but if one day someone types `<RegionSelect component="p" />`, the console error reappears for that page

```suggestion
    <Box {...rest} component="span" sx={{ display: 'block' }}>
```

Reply:
Yea, placing `{...rest}` before ensures custom props override passed user props, while still allowing the  passed `sx` prop to function correctly.

fixed in  8c9cf8afd12a02a37509ed1e92eab15f2534614c